CalibPPS/ESProducers/plugins/WriteTotemDAQMapping.cc

```diff
@@ -0,0 +1,67 @@
+/****************************************************************************
+ *
+ ****************************************************************************/
+
+#include "CondFormats/DataRecord/interface/TotemAnalysisMaskRcd.h"
+#include "CondFormats/DataRecord/interface/TotemReadoutRcd.h"
+#include "CondFormats/PPSObjects/interface/TotemAnalysisMask.h"
+#include "CondFormats/PPSObjects/interface/TotemDAQMapping.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include <fstream>
+#include <iostream>
+
+//----------------------------------------------------------------------------------------------------
+
+/**
+ *\brief Writes to file the DAQ mapping loaded by TotemDAQMappingESSourceXML.
+ **/
+class WriteTotemDAQMapping : public edm::one::EDAnalyzer<> {
+public:
+  WriteTotemDAQMapping(const edm::ParameterSet &ps);
+  ~WriteTotemDAQMapping() override {}
+
+private:
+  /// label of the CTPPS sub-system
+  std::string subSystemName;
+  std::ofstream outStream;
+  const edm::ESGetToken<TotemDAQMapping, TotemReadoutRcd> mappingToken;
+  const edm::ESGetToken<TotemAnalysisMask, TotemAnalysisMaskRcd> maskToken;
```

Comment:
thanks for having this `const`, the trailing `_` in the end would be appreciated too